package.json

```diff
@@ -43,20 +43,20 @@
   },
   "homepage": "https://github.com/brandonroberts/ngrx-store-freeze#readme",
   "peerDependencies": {
-    "@ngrx/store": ">=4.0.0"
+    "@ngrx/store": ">=4.0.0 | 6.0.0-beta.1"
```

Comment:
There should be 2 pipes here and `^6.0.0-beta.1`. That way once 6.0.0 lands, we won't need to change this immediately.

Reply:
You;re right, i wasn't sure if with the caret, it would accept versions other than 6.0.0-[some beta/alpha version]. Should've looked it up. And i thought a single pipe would work as well, i've added it in the next push.